resources/quality/Infinity3D/Infinity3D_fast.inst.cfg

```diff
@@ -0,0 +1,66 @@
+[general]
+version = 1
+name = Fast
+definition = Infinity3d
+
+[metadata]
+setting_version = 20
+type = quality
+quality_type = fast
+weight = -2
+global_quality = True
+
+[values]
+layer_height = 0.4
+layer_height_0 = 0.3
+initial_layer_line_width_factor = 100
```

Comment:
```suggestion
```
The setting already is a 100, no need to define it. It's better not to re-define it. As a rule of thumb, profiles should only contain what they change.